README.md

```diff
@@ -2,8 +2,8 @@
 
 | lib to simulate  | version | path builded in application                  |
 | ---------------- | ------- | -------------------------------------------- |
-| Quantum          | 9.0.1   | node_modules/@catho/Quantum                  |
-| Catho-components | 5.8.2  | node_modules/@catho-private/catho-components |
+| Quantum          | 9.2.0   | node_modules/@catho/Quantum                  |
```

Comment:
```suggestion
| Quantum          | 9.4.0   | node_modules/@catho/Quantum
```

9.4.0  Wouldn't it be the current version?

Reply:
At the time the quantum version was updated this was the latest version. There was a merger in quantum and now it is outdated. In order to make this as standardized as possible with the performance tests, I think it's interesting to keep the quantum in the 9.1.1 version that was used in the test. I will adjust